### PR TITLE
GH38823: Fixing typo for namespace selector in 4.8 RN

### PR DESCRIPTION
**Pull request by ctauchen**
Fixes syntax error in namespace selector. `=` should be `:`

See also https://github.com/openshift/openshift-docs/pull/38897 

Preview: https://deploy-preview-38898--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-8-release-notes#ocp-4-8-network-policy-host-network-ingress-controllers

cc @jboxman

### CODE REVIEW

File: release_notes/ocp-4-8-release-notes.adoc [hunk 1]
```diff
@@ -406,19 +406,19 @@ For more information, refer to xref:../networking/openshift_sdn/assigning-egress
 ==== Network policy supports selecting host network Ingress Controllers
 
 When using the OpenShift SDN or OVN-Kubernetes cluster network providers, you can select traffic from Ingress Controllers in a network policy rule regardless of whether an Ingress Controller runs on the cluster network or the host network.
-In a network policy rule, the `policy-group.network.openshift.io/ingress=""` namespace selector label matches traffic from an Ingress Controller. You can continue to use the `network.openshift.io/policy-group: ingress` namespace selector label, but this is a legacy label that can be removed in a future release of {product-title}.
+In a network policy rule, the `policy-group.network.openshift.io/ingress:""` namespace selector label matches traffic from an Ingress Controller. You can continue to use the `network.openshift.io/policy-group: ingress` namespace selector label, but this is a legacy label that can be removed in a future release of {product-title}.
```
Comment (jboxman-rh) [inline release_notes/ocp-4-8-release-notes.adoc:409]: @ctauchen afaik a space is permissible here.

File: release_notes/ocp-4-8-release-notes.adoc [hunk 2]
```diff
@@ -406,19 +406,19 @@ For more information, refer to xref:../networking/openshift_sdn/assigning-egress
...
+- A cluster that uses the OpenShift SDN cluster network provider could select traffic from an Ingress Controller on the host network only by applying the `network.openshift.io/policy-group: ingress` label to the `default` namespace.
 - A cluster that uses the OVN-Kubernetes cluster network provider could not select traffic from an Ingress Controller on the host network.
 
 For more information, refer to xref:../networking/network_policy/about-network-policy.adoc#about-network-policy[About network policy].
 
 [id="ocp-4-8-network-policy-host-network-policy-group"]
 ==== Network policy supports selecting host network traffic
 
-When using either the OVN-Kubernetes cluster network provider or the OpenShift SDN cluster network provider, you can use the `policy-group.network.openshift.io/host-network: ""` namespace selector to select host network traffic in a network policy rule.
+When using either the OVN-Kubernetes cluster network provider or the OpenShift SDN cluster network provider, you can use the `policy-group.network.openshift.io/host-network:""` namespace selector to select host network traffic in a network policy rule.
```
Comment (jboxman-rh) [inline release_notes/ocp-4-8-release-notes.adoc:421]:
```suggestion
When using either the OVN-Kubernetes cluster network provider or the OpenShift SDN cluster network provider, you can use the `policy-group.network.openshift.io/host-network: ""` namespace selector to select host network traffic in a network policy rule.
```

File: release_notes/ocp-4-8-release-notes.adoc (hunk 1 above)
Comment (ctauchen) [reply release_notes/ocp-4-8-release-notes.adoc]: Searched the docs, `: ""` is clearly more common. Changed. Thanks for spotting that!

File: release_notes/ocp-4-8-release-notes.adoc (hunk 2 above)
Comment (ctauchen) [reply release_notes/ocp-4-8-release-notes.adoc]: Agreed, changed.

**Comment by ctauchen** [comment]
@asood-rh Can you take a quick look at this?

**Comment by lpettyjo** [comment]
Peer review: LGTM

**Pull request merged**